apm-agent-core/src/test/java/co/elastic/apm/agent/MockReporter.java

```diff
@@ -97,6 +98,8 @@ public class MockReporter implements Reporter {
 
     private boolean closed;
 
+    private static final long DEFAULT_ASSERTION_TIMEOUT = OS.WINDOWS.isCurrentOs() ? 3000 : 1000;
```

Comment:
[minor] just add a comment to help future codebase archeology on the intent here.
```suggestion
    // we have to use a longer timeout on Windows to help reduce flakyness
    private static final long DEFAULT_ASSERTION_TIMEOUT = OS.WINDOWS.isCurrentOs() ? 3000 : 1000;
```